### PR TITLE
WIP: New team page

### LINKED ISSUE #40: Highlight project maintainers on community page
Lets's highlight sub-area maintainers on the Community page.

## Proposal

In https://github.com/storybooks/storybook/pull/6509 I added a config file `automention.yml` that maps github labels to maintainers who are willing to review PRs and provide user support around specific project areas, ranging from view layers such as `react`/`vue`/`angular` to addons like `knobs`/`storysource` to features like `typescript`/`theming`.

Proposal:
- Select a subset of these areas to highlight
- Read latest `automention.yml`
- Display avatars/profile links for the maintainers in each area

**NOTE: needs design** @domyen

### PR DESCRIPTION
**Pull request by kylesuss**
Building off of #78 

There are a few missing pieces still:
- [x] Need stories for all of the `TeamScreen` components
- [x] Need to fill in the maintainers (currently just placeholders)
- [x] Need a github token for the netlify build

Closes #40

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ version: 2
 jobs:
   build:
     docker:
-      - image: circleci/node:8.12
+      - image: circleci/node:12.14.1
```
Comment (kylesuss) [inline .circleci/config.yml]: A nested dependency required >=10, so I figured I would upgrade to the latest LTS version.

**Comment by darleendenno** [comment]
@domyen @kylesuss where are we on this? I've checked the preview and it's looking good 👀

**Comment by darleendenno** [comment]
@shilman what are your thoughts on this? is there a way we can get this over the finish line?

**Comment by shilman** [comment]
Yes!!! I've created a bunch of GitHub teams over the past year-- let me take a gander at how it shows up in frontpage

**Pull request closed without merging**